components/blocks/FilesToCards.vue

```diff
@@ -151,24 +151,49 @@
           width="medium"
         >
           <template #header>
-            <span v-if="item.group" class="radius-0 tag is-link has-text-light"
-              ><abbr :title="item.group | expandAcronym">
-                {{ item.group }}
-              </abbr></span
+            <div v-if="item.tags"
+              ><abbr class="radius-0 tag is-link has-text-light m-1" v-for="tag in item.tags" :key="tag" :title="tag | expandAcronym">
```

Comment:
the method of `val | filterFunction` has been deprecated in vue 3, so I would avoid using it generally.  instead just make expandAcronym a method and then `filterFunction(val)`

Reply:
i dont actually think expandAcronym is doing anything. im going to delete it. it is used this way in the cardGroup component. will change it for deprecation reasons.